packages/reports/addon/components/filter-builders/base.hbs

```diff
@@ -13,8 +13,14 @@
     <div class="filter-builder__subject xs-col-1-2 sm-col-1-2 col-1-5">
       <div class="filter-builder__subject-name">
         <span class="name m-r-4">{{this.columnName}}</span>
-        {{#each this.paramValues as |parameter|}}
-          <DenaliChip class="m-r-4" @size="small">{{parameter}}</DenaliChip>
+        {{#each this.paramValues as |parameter index|}}
+          <DropdownParameterPicker 
+            @filter={{@filter}} 
```

Comment:
Instead of passing the whole filter and looking up the specific param, can we pass down the param key and value as args?

Reply:
I would have to move 4 getters to filter-builder/base, you'd prefer that over whats currently happening?